### ISSUE TITLE
nitropy-0.4.32.exe missing module

### ISSUE THREAD

**Issue by hase77** [Nitrokey/pynitrokey#314]
New release of nitropy version 0.4.32 for windows seems to be missing a module:

```
PS C:\Users\ms\Desktop> .\nitropy-0.4.32.exe
Traceback (most recent call last):
  File "requests\compat.py", line 11, in <module>
ModuleNotFoundError: No module named 'chardet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "nitropy.py", line 13, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\__init__.py", line 21, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\fido2.py", line 35, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\update.py", line 18, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\__init__.py", line 45, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\exceptions.py", line 9, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\compat.py", line 13, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\__init__.py", line 24, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\api.py", line 5, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\cd.py", line 9, in <module>
ModuleNotFoundError: No module named 'charset_normalizer.md__mypyc'
[3296] Failed to execute script 'nitropy' due to unhandled exception!
```

**Comment by robin-nitrokey** [comment]
This seems to be an issue with PyInstaller: https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/534

It looks like we should be able to fix it by ensuring `"pyinstaller-hooks-contrib>=2022.15"` though I don’t see where PyInstaller is installed at the moment.

**Comment by robin-nitrokey** [comment]
It comes from the `tobix/pywine` Dockerfile and is installed with pip inside wine.  We can probably fix this by updating `pyinstaller-hooks-contrib` in [`build-wine-docker.sh`](https://github.com/Nitrokey/pynitrokey/blob/master/wine-build/build-wine-docker.sh).

**Comment by Ryeera** [comment]
Bump! Having the same issue running on Windows 11.

**Comment by ableeker** [comment]
With version 0.4.32 I get the exact same message as hase77 running Windows 11.

Running Linux with wine and docker I've created a new version of 0.4.32 for Windows which included the fix proposed by robin-nitrokey. But when I run this version I get the following error:
```
C:\Users\ableeker\Downloads>nitropy-0.4.32a.exe --version
Traceback (most recent call last):
  File "nitropy.py", line 13, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\__init__.py", line 22, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\nethsm.py", line 19, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\nethsm\__init__.py", line 18, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\nethsm\client\__init__.py", line 17, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\nethsm\client\api_client.py", line 29, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "frozendict\__init__.py", line 33, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\ableeker\\AppData\\Local\\Temp\\_MEI89122\\frozendict\\VERSION'
[9932] Failed to execute script 'nitropy' due to unhandled exception!
```

**Comment by ableeker** [comment]
I noticed there's now a new version of nitropy: 0.4.33. However when I ran this version, I got the following result.
```
D:\>nitropy-0.4.33.exe
Traceback (most recent call last):
  File "requests\compat.py", line 11, in <module>
ModuleNotFoundError: No module named 'chardet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "nitropy.py", line 13, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\__init__.py", line 24, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\fido2.py", line 35, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pynitrokey\cli\update.py", line 18, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\__init__.py", line 45, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\exceptions.py", line 9, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "requests\compat.py", line 13, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\__init__.py", line 24, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\api.py", line 5, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "charset_normalizer\cd.py", line 9, in <module>
ModuleNotFoundError: No module named 'charset_normalizer.md__mypyc'
[4204] Failed to execute script 'nitropy' due to unhandled exception!
```

**Comment by christianwicke** [comment]
I got the same error with 0.4.34 as ableeker in the last comment.

**Comment by RcTomcat1** [comment]
Issue still persists.

**Comment by daringer** [comment]
Yes, we have overthrown the old windows build and are working on a new one, which will also include an `.msi`... Will be available soon(tm)

**Comment by robin-nitrokey** [comment]
The new binary and installer are available since v0.4.36 and should fix this problem.
